lib/bosh/workspace/project_deployment.rb

```diff
@@ -63,14 +54,34 @@ def execute_stub(stub_file)
 
     private
 
+    def stubs_dir
+      File.expand_path(File.join(file_dirname, "../stubs"))
+    end
+
+    def stub_file
+      Dir[File.join(stubs_dir, "#{file_basename('.*')}*")].first
```

Comment:
hey, @rkoster. 
I am not sure about this changes. What if I want to store several stub files in stub folder? What if I use them for different deployments?